src/web/resources/report.css

```diff
@@ -80,7 +101,7 @@ tr.uni-start  td:nth-child(3) {background-color:#ff5e3a;color:#f7f7f7;}
 tr.ex-start   td:nth-child(3) {background-color:#e0f8d8;}
 
 tr.gt-target  td:nth-child(3) {background-color:#87fc70;}
-tr.gt-target  td:nth-child(4) {background-color:#0bd318;}
+tr.gt-target  td:nth-child(4) {background-color:#87fc70;}
```

Comment:
Why? I'm not thrilled with how the "target" column works at all, but the idea is that the "Result" column is highlighted to show how result compares to initial program, while the "Target" column is highlighted to show how result compares to target.